src/yoroi-wallets/cardano/types.ts

```diff
@@ -100,8 +100,14 @@ export interface WalletInterface {
 
   save(): Promise<void>
 
+  remove(): Promise<void>
```

Comment:
whats the difference between `remove()` and `clear()`?

Reply:
When we remove the wallet from `walletManager` we clear the wallet and now we remove the memos too.

On resync the wallet is cleared but the memos are not removed.

So I think we have to make a distinction here.

Also maybe is a good starting point to move all the storage removing stuff from 'walletManager.removeWallet' to the wallet itself, is it ok?

Reply:
eventually, not this pr